py/dml/codegen.py

```diff
@@ -3047,8 +3062,14 @@ def mark_method_exported(func, name, export_site):
         (otherfunc, othersite) = exported_methods[name]
         report(ENAMECOLL(export_site, othersite, name))
     else:
+        if (export_site.dml_version() != (1, 2)
```

Comment:
remove 1.2 precond? a 1.2 dev can import a 1.4 file

Reply:
All these checks are because `method extern`s are exported differently (see c_backend), and the way to check for that is to check for the `dml_version()` at the export site.

Reply:
Right. Perhaps add a comment that the static export is needed because generate_extern_trampoline relies on a static trampoline.

Reply:
Done.